test/es2015/module-support-assessment-test.mjs

```diff
@@ -2,13 +2,54 @@
 import referee from "referee";
 import sinon from "../../lib/sinon";
 import * as aModule from "./a-module";
-const {assert} = referee;
-
-describe("Explicit lack of support for stubbing Modules", function() {
-    it("should give a proper error message for modules without default export", function() {
-        assert.exception(function() {
-            sinon.stub(aModule, "anExport").returns(400);
-        },
-        /TypeError: No support for stubbing ES Modules/);
+import aModuleWithDefaultExport from "./a-module-with-default";
+import functionModule, * as functionModuleAlternative from "./a-function-module";
+const {assert, refute} = referee;
+
+function createTestSuite(action){
+    var stub;
+    var errorRegEx = /TypeError: ES Modules cannot be (stubbed|spied)/;
+
+    afterEach(function() {
```

Comment:
This seems duplicated with the same `afterEach` inside the `describe` below ... is that intentional?